python_modules/dagster-graphql/dagster_graphql/client/client.py

```diff
@@ -131,11 +131,11 @@ def _core_submit_execution(
         check.str_param(pipeline_name, "pipeline_name")
         check.opt_str_param(mode, "mode")
         check.opt_str_param(preset, "preset")
-        run_config = check.opt_dict_param(run_config, "run_config")
+        _run_config = check.opt_dict_param(run_config, "run_config")
```

Comment:
This was throwing a mypy error until I switched it to `opt_mapping_param`. I believe it's because the return type of `opt_dict_param` isn't a (strict?) subset of `run_config`s original type, so it was complaining that the variable type was reassigned.